bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -184,6 +184,7 @@ fn build_function_arn(account_id: &str, region: &str, function_name: &str) -> St
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    let start_time = Instant::now();
```

Comment:
We also do `Instant::now()` in `load_configs` for `aws_config` sandbox init start, maybe worth refactoring that if it's going to be used here? Either we use this one and make the `aws_config` to have it available, or we use the one the `aws_config` has, do as you prefer, another alternative could be possible!

Reply:
yup good point, combined.